packages-native/animation/package.json

```diff
@@ -1,45 +1,39 @@
-    },
-    "packagePath": "com.mendix.widget.native",
-    "scripts": {
-        "start": "..\"/../node_modules/.bin/pluggable-widgets-tools\" start:ts:native -- --subprojectPath packages-native/animation",
-        "build": "..\"/../node_modules/.bin/pluggable-widgets-tools\" build:ts:native --subprojectPath packages-native/animation",
-        "release": "..\"/../node_modules/.bin/pluggable-widgets-tools\" release:ts:native --subprojectPath packages-native/animation",
-        "lint": "..\"/../node_modules/.bin/eslint\" --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
-        "test": "..\"/../node_modules/.bin/jest\" --config ../../scripts/test/jest.native.config.js",
-        "version": "npm run release",
-        "postversion": "..\"/../node_modules/.bin/ts-node\" ../../scripts/release/UpdateGithubRelease.ts --project ../../scripts/tsconfig.json"
-    },
-    "jest-junit": {
-        "output": "./dist/testresults/TESTS-Jest.xml"
-    },
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/mendix/widgets-resources.git"
-    },
-    "license": "Apache-2.0",
-    "devDependencies": {
-        "@types/big.js": "^4.0.5",
-        "@types/classnames": "^2.2.6",
-        "@types/enzyme": "^3.1.14",
-        "@types/jest": "^24.0.0",
-        "@types/react": "^16.8.17",
-        "@types/react-dom": "^16.8.4",
-        "@types/react-native": "^0.57.56"
-    },
-    "dependencies": {
-        "big.js": "^5.2.2",
-        "react-native-animatable": "^1.3.2",
-        "@native-mobile-resources/util-widgets": "^1.0.0",
-        "@widgets-resources/piw-utils": "^1.0.0"
-    }
-}
+  "name": "animation",
+  "widgetName": "Animation",
+  "version": "1.0.0",
+  "description": "Container for animations",
+  "copyright": "Mendix 2019",
+  "author": "Widget Team",
+  "config": {
+    "widgetPath": "../test-project/mxproject/widgets",
+    "projectPath": "../test-project/mxproject"
+  },
+  "packagePath": "com.mendix.widget.native",
+  "scripts": {
+    "start": "pluggable-widgets-tools start:ts:native -- --subprojectPath packages-native/animation",
+    "build": "pluggable-widgets-tools build:ts:native --subprojectPath packages-native/animation",
+    "release": "pluggable-widgets-tools release:ts:native --subprojectPath packages-native/animation",
+    "lint": "eslint --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
+    "test": "jest --config ../../scripts/test/jest.native.config.js",
+    "version": "npm run release",
+    "postversion": "ts-node ../../scripts/release/UpdateGithubRelease.ts --project ../../scripts/tsconfig.json"
+  },
+  "jest-junit": {
+    "output": "./dist/testresults/TESTS-Jest.xml"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendix/widgets-resources.git"
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@mendix/pluggable-widgets-tools": "8.9.0"
+  },
+  "dependencies": {
+    "big.js": "^5.2.2",
+    "react-native-animatable": "^1.3.2",
```

Comment:
Should we mark this one as external in PIW-Tools?

Reply:
no, it is not shipped with client